lib/aliased.pm

```diff
@@ -196,6 +196,14 @@ use the new module by only changing one line of code.
   use aliased "New::Module::Name" => "Old::Module::Name";
   my $thing = Old::Module::Name->new;
 
+If you in two different packages alias like in the above example and use the
```

Comment:
The grammar is awkward here; I'm not quite sure what you mean?  perhaps "If you use two different package aliases.."?